src/sfizz/Synth.cpp

```diff
@@ -844,13 +844,18 @@ void sfz::Synth::noteOnDispatch(int delay, int noteNumber, float velocity) noexc
 }
 
 void sfz::Synth::cc(int delay, int ccNumber, uint8_t ccValue) noexcept
+{
+    const auto normalizedCC = normalizeCC(ccValue);
+    hdcc(delay, ccNumber, normalizedCC);
+}
+
+void sfz::Synth::hdcc(int delay, int ccNumber, float normValue) noexcept
 {
     ASSERT(ccNumber < config::numCCs);
     ASSERT(ccNumber >= 0);
-    const auto normalizedCC = normalizeCC(ccValue);
 
     ScopedTiming logger { dispatchDuration, ScopedTiming::Operation::addToDuration };
-    resources.midiState.ccEvent(delay, ccNumber, normalizedCC);
+    resources.midiState.ccEvent(delay, ccNumber, normValue);
```

Comment:
The `normalizeCC` function clamps the value. Should we do so here too? Or ignore oob values?

Reply:
We should do like ARIA. I think we need to look at behavior and replicate the same.
`set_hdcc200=2.0`

Reply:
It doesn't clamp, so let's not clamp either.